src/modules/infobox/components/styling/DrawingStyleMediaLink.vue

```diff
@@ -0,0 +1,132 @@
+    } else {
+        return `<iframe src="${generatedMediaLink.value}" height="200" width="auto"></iframe>`
+    }
+}
+function createImage() {
+    return `<image src="${generatedMediaLink.value}" style="max-height:200px;"/>`
+}
+function createLink() {
+    if (linkDescription.value) {
+        return `<a target="_blank" href="${generatedMediaLink.value}">${linkDescription.value}</a>`
+    } else {
+        return `<a target="_blank" href="${generatedMediaLink.value}">${generatedMediaLink.value}</a>`
+    }
+}
+function addLink(generatedMediaLink) {
+    if (!urlValid.value) {
+        return
+    }
+    switch (mediaType.value) {
+        case MediaTypes.link:
+            emit('generatedMediaLink', createLink(generatedMediaLink))
+            break
+        case MediaTypes.image:
+            emit('generatedMediaLink', createImage(generatedMediaLink))
+            break
+        case MediaTypes.video:
+            emit('generatedMediaLink', createVideo(generatedMediaLink))
+            break
+    }
+    clearInput()
+}
+function clearInput() {
+    generatedMediaLink.value = null
+}
+function validateUrl(url) {
+    if (!url) {
+        return 'no_url'
+    } else if (!isValidUrl(url)) {
+        return 'invalid_url'
+    }
+    return ''
+}
+</script>
+
+<template>
+    <div class="px-3 pb-2">
+        <div v-if="descriptionLabel" class="pb-2">
+            <label class="form-label" for="drawing-style-media-link-description">
+                {{ descriptionLabel }}
+            </label>
+            <div class="input-group d-flex needs-validation">
+                <TextInput
+                    id="drawing-style-media-link-description"
+                    v-model="linkDescription"
+                    class="feature-url-description mb-2"
+                    :placeholder="i18n.t('link_description')"
+                    data-cy="drawing-style-media-description-input"
+                />
+            </div>
+        </div>
+        <label class="form-label" for="drawing-style-media-url-description">
+            {{ urlLabel }}
+        </label>
+        <div class="d-flex align-items-start">
+            <TextInput
+                id="drawing-style-media-url-description"
+                v-model="generatedMediaLink"
+                class="feature-url-description mb-2"
+                :placeholder="i18n.t('paste_url')"
+                :validate="validateUrl"
+                :is-last-element="false"
+                data-cy="drawing-style-media-url-input"
+                @keydown.enter="addLink(generatedMediaLink)"
+            />
+            <button
+                :disabled="!urlValid"
+                class="btn btn-default btn-outline-secondary rounded-0 rounded-end"
+                type="button"
+                data-cy="drawing-style-media-generate-button"
+                @click="addLink(generatedMediaLink)"
+            >
+                {{ i18n.t('add') }}
+            </button>
```

Comment:
Here the `Add` button should be put inside the bootstrap `input-group` of the `TextInput` using slot. This way in theory you don't need the extra property `is-last-element` as bootstrap should handle rounded corner correctly. However the last point is not always true, but I would still use slot as it is more logical to have all buttons inside the input group.